test/unittest/xmlparser/XMLProfileParserTests.cpp

```diff
@@ -29,6 +29,10 @@ using namespace eprosima::fastrtps;
 using namespace eprosima::fastrtps::rtps;
 using namespace ::testing;
 
+using eprosima::fastdds::dds::Log;
+using eprosima::fastdds::dds::LogMock;
+using eprosima::fastdds::dds::LogConsumer;
+
 // Initialize Log mock
 LogMock* log_mock;
 std::function<void(std::unique_ptr<LogConsumer>&&)> Log::RegisterConsumerFunc =
```

Comment:
The build errors on VS2015 can be solved by creating the lambdas before this line (with auto), and assigning them later

Reply:
I've actually decided to make the lambdas disappeared by defining suitable functions on the source files.